.github/workflows/terraform-lint.yml

```diff
@@ -0,0 +1,14 @@
+name: terraform-lint
+
+on: [pull_request]
+
+jobs:
+  terraform-lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@master
+    - name: Lint Terraform
+      uses: actionshub/terraform-lint@main
```

Comment:
One for the future, but I wonder if it's worth looking at using [the official Actions instead](https://github.com/hashicorp/setup-terraform)?